site/src/theme/constants.ts

```diff
@@ -1,7 +1,7 @@
 export const borderRadius = 8;
 export const MONOSPACE_FONT_FAMILY =
   "'IBM Plex Mono', 'Lucida Console', 'Lucida Sans Typewriter', 'Liberation Mono', 'Monaco', 'Courier New', Courier, monospace";
-export const BODY_FONT_FAMILY = `"Inter", system-ui, sans-serif`;
+export const BODY_FONT_FAMILY = `"Inter Variable", system-ui, sans-serif`;
```

Comment:
Just to be on the safe side, I feel like it'd be good to include Inter as the second variable in the stack
We won't be serving it up, but if the user just happens to have it but the not the variable version downloaded, they'll still get a pretty equivalent UI

Reply:
I think it is too much to be honest since this font will be cached on the first load.